### ISSUE TITLE
DTI_CMH_S002_01_01

### ISSUE THREAD

**Issue by navonacalarco** [TIGRLab/admin#1296]
Find out why participant has a second timepoint with data, but first timepoint has none.

**Comment by josephdviviano** [comment]
I think this subject's 01 scan will be found in `DTI15T`

**Comment by navonacalarco** [comment]
Do you see it there? I don't.

I've aiming to address this and my issues tomorrow morning - appreciate the gesture to assist, but not necessary.

**Comment by josephdviviano** [comment]
Good point, I misread this one. You're going way back in time for this. Will need to track down `DTI_TGH_S002_01` . I'm not sure where the records would be for these. This is the Rotenberg era.

**Comment by josephdviviano** [comment]
If the data exists you will be able to track it down using the info in `/archive/data/DTI15T/metadata/scans.csv` and `/archive/data/DTI15T/metadata/raw`

**Comment by josephdviviano** [comment]
```

DTI_ID | Initials | DateOfBirth | Scan Date | Target Name | Path
06S002 | IA | 06-05-29 | 03-05-07 | DTI_TGH_S002_01_01 | pt_06S002


```

**Comment by navonacalarco** [comment]
@josephdviviano @DESm1th Finally looking at this. Sorry for delay.

I can't find the scan.

However, as Joseph indicated above, `DTI_Scan_Info_24_Sept_2015.xls` found in /archive/data/DTI15T/metadata/raw states the scan was completed on  03-05-07. And what's more odd still is that this spreadsheet was created by @edickie  and I just two years ago: and the fact that this entry exists on this sheet means that we had it at that point in time.

I'm at a loss as to where the scan has gone. I took a cursory look through the rest of the files, and it doesn't look like anything else is missing.

What should we do?

**Comment by navonacalarco** [comment]
After conversation with @josephdviviano and @edickie, we think that if it ever existed, we unfortunately don't have it now; could have been a casualty of RAID4, or perhaps something else.

**Comment by josephdviviano** [comment]
@navonacalarco wait -- the RAID 4 server still has everything.

Can you compile a list of  `.zip` files we're missing (using the original names i.e. `pt_blahblah.zip`) and I can get those to Peter Bloomfield downstairs.

**Comment by navonacalarco** [comment]
Great! Thanks.

**Comment by josephdviviano** [comment]
@navonacalarco just a reminder that this is outstanding.

**Comment by navonacalarco** [comment]
Yep, thanks. Hope to get to it this week.
